mods/seenmc.py

```diff
@@ -0,0 +1,44 @@
+import time
+import datetime
+import config
+
+from common import *
+RegisterMod(__name__)
+
+def Parse(raw, text):
+	match = re.match("^:(.[^!]+)![^@]+@[^ ]+ PRIVMSG (#[^ ]+) :(.+)", raw)
```

Comment:
To look for Minecraft messages, use the regex here:
https://github.com/jacob1/jacobot/blob/master/common.py#L43